pkgs/applications/science/biology/jellyfish/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, lib }:
+
+stdenv.mkDerivation rec {
+    pname = "jellyfish";
+    version = "2.3.0";
+
+    src = fetchurl {
+        url = "https://github.com/gmarcais/Jellyfish/releases/download/v${version}/${pname}-${version}.tar.gz";
```

Comment:
Why not use fetchFromGitHub here?

Reply:
@Synthetica9 I tried but it didn't work (I forgot what the error was, can rerun it again and post it here if needed)

Reply:
Please do.

Reply:
@Synthetica9 oh I remembered. I always get hash mismatch. How do you get the right hash? I did:
```
find path/to/folder -type f -print0 | sort -z | xargs -0 sha256sum | sha256sum
```

Reply:
I just insert a wrong SHA256 hash (all 0's) and copy the correct hash from the error message, it's called TOFU (trust on first use)

Reply:
@Synthetica9 ah I tried that before, but I copied the `wanted` instead of the `got` :facepalm: . Thanks!